src/nvim/if_cscope.c

```diff
@@ -2072,22 +2070,21 @@ static char *cs_resolve_file(int i, char *name)
  */
 static int cs_show(exarg_T *eap)
 {
-  short i;
   if (cs_cnt_connections() == 0)
     MSG_PUTS(_("no cscope connections\n"));
   else {
     MSG_PUTS_ATTR(
         _(" # pid    database name                       prepend path\n"),
         hl_attr(HLF_T));
-    for (i = 0; i < csinfo_size; i++) {
+    for (size_t i = 0; i < csinfo_size; i++) {
       if (csinfo[i].fname == NULL)
         continue;
 
       if (csinfo[i].ppath != NULL)
-        (void)smsg("%2d %-5ld  %-34s  %-32s",
+        (void)smsg("%2zu %-5ld  %-34s  %-32s",
             i, (long)csinfo[i].pid, csinfo[i].fname, csinfo[i].ppath);
       else
-        (void)smsg("%2d %-5ld  %-34s  <none>",
+        (void)smsg("%2zu %-5ld  %-34s  <none>",
```

Comment:
Sorry I didn't notice this, but the `%-5ld` should be changed to `"%-5" PRId64`.

`%ld` is not portable.